engine/apps/slack/views.py

```diff
@@ -578,7 +578,10 @@ def post(self, request):
             if slack_team_identity is not None:
                 clean_slack_integration_leftovers.apply_async((organization.pk,))
                 if settings.FEATURE_MULTIREGION_ENABLED:
-                    delete_slack_connector(str(organization.uuid))
+                    if settings.CHATOPS_V3:
+                        unlink_slack_team(str(organization.uuid), slack_team_identity.slack_id)
+                    else:
+                        delete_slack_connector(str(organization.uuid))
```

Comment:
maybe it'd be better to move this `if settings.CHATOPS_V3` logic to `delete_slack_connector` instead? the same goes for `check_slack_installation_possible`, `create_slack_connector`, `create_oncall_connector`, `delete_oncall_connector`, etc. 
This way a caller wouldn't need to know if it's chatops v3 or not, wdyt?

Reply:
Implemented, thanks!